00-intro-carto/exercises/intro.md

```diff
@@ -0,0 +1,63 @@
+# CARTO BUILDER, Dashboards, Dataset View and Public Profile Introduction
+
+## Contents
+
+0. [Resources](#resources)
+1. [Maps Dashboard](#maps)
+2. [Datasets Dashboard](#datasets)
+3. [Dataset View](#dataset-view)
+4. [BUILDER](#builder)
+5. [Public Profile](#profile)
+
+### 0. Resources <a name="resources"></a>
+
+* [CARTO Documentation](https://carto.com/docs/)
+* [LEARN Guides](https://carto.com/learn/guides)
+
+### 1. Maps Dashboard <a name="maps"></a>
+
+* URL: `https://{{USER_NAME}}.carto.com/dashboard/dataset`
+
+![maps-dashboard](imgs/maps-dashboard.png)
+
+* CARTO Maps Dashboard shows BUILDER maps ordered by `date` (default), `visits` or `likes`, users can `SEARCH` and switch to the maps with `likes`. 
```

Comment:
I recommend that you use the UI name of the feature, for consistency across all documentation resources. _Your maps_ dashboard or _Your datasets_ dashboard, etc. (related learn guide: https://carto.com/learn/guides/intro/getting-started-with-carto-builder) and/or Internal Wiki (https://github.com/CartoDB/learn/wiki/Builder-UI-Terminology#carto-dashboard) about UI

Reply:
Now after reading the guide and internal wiki, I do not know if make sense to have also this "exercise". @jsanz what do you think? Tomorrow I can do it without this.

Reply:
sorry @ramiroaznar but I don't have context, can you please elaborate more your question?

Reply:
Having a look at the guide and wiki Christa has shared, I do not if this exercise can be seen as duplicate work. It's always better to point to the "official" resources...